velox/exec/Driver.cpp

```diff
@@ -856,6 +931,12 @@ std::string Driver::toJsonString() const {
   obj["closed"] = closed_.load();
   obj["queueTimeStartMicros"] = queueTimeStartMicros_;
   obj["curOpIndex"] = curOpIndex_;
+  const auto ocs = opCallStatus();
+  if (!ocs.empty()) {
+    obj["curOpCall"] =
```

Comment:
It might be confusing when curOpIndex doesn't math curOpCall. I see that CALL_OPERATOR is sometimes called with curOpIndex and other times with curOpIndex + 1.

Reply:
@mbasmanova Life is confusing. :)
What is your suggestion?

Reply:
Will replace "curOpIndex" by "curOp" where the operator type would be.